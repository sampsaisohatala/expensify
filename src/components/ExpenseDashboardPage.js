import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
   <div>
      <h2>Dashboard</h2>
      <ExpenseListFilters />
      <ExpenseList />
   </div>
);

export default ExpenseDashboardPage;
