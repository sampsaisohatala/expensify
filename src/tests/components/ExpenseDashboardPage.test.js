import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

Enzyme.configure({
   adapter: new Adapter(),
});

test('should render ExpenseDashboardPage correctly', () => {
   const wrapper = shallow(<ExpenseDashboardPage />);
   expect(toJSON(wrapper)).toMatchSnapshot();
});
