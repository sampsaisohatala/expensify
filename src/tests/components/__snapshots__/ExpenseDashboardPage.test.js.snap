// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correctly 1`] = `
<div>
  <h2>
    Dashboard
  </h2>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
